client/.env.development

```diff
@@ -9,3 +9,5 @@ REACT_APP_SCANS_SERVICE_RESTART=http://localhost:4000/api/v1/scans/{0}/restart/
 REACT_APP_SCANS_SERVICE_RESULTS=http://localhost:4000/api/v1/scans/{0}/results/
 
 REACT_APP_SOURCES_SERVICE=http://localhost:4000/api/v1/sources/
+
+REACT_APP_DEBUG_MIDDLEWARE=false
```

Comment:
Note: Not sure this is the best location for this setting... Something that would limit it on a per dev basis feels better. In a follow up PR I'll look at removing this and updating the ReadMe to reflect that:
>"[if you want to run the logger create a .env.local file with REACT_APP_DEBUG_MIDDLEWARE=true]"